src/sage/schemes/generic/divisor_group.py

```diff
@@ -209,10 +209,10 @@ def _an_element_(self):
 
         EXAMPLES::
 
-            sage: A.<x, y> = AffineSpace(2, CC)
-            sage: C = Curve(y^2 - x^9 - x)
+            sage: A.<x, y> = AffineSpace(2, CC)                                         # needs sage.rings.real_mpfr
+            sage: C = Curve(y^2 - x^9 - x)                                              # needs sage.rings.real_mpfr sage.schemes
             sage: from sage.schemes.generic.divisor_group import DivisorGroup
```

Comment:
Ah, now I see curves are in `sage.schemes`. So dynamical systems are also in `sage.schemes`.